config/locales/en.yml

```diff
@@ -111,179 +111,139 @@ en:
-          online services
-        paragraph: Use data science, data analytics and research techniques to make
-          services better and more efficient
+          - text: How GOV.UK can support the UK's recovery from coronavirus through more responsive and joined-up guidance and services
+          - text: Improve post-Brexit content and navigation, such as for borders and trade
+          - text: Use the 'starting a business' work as a model for simplifying other complex, whole user journeys
+          - text: Test a prototype business hub page for different types and stages of business
+      - heading: Connect insights across GOV.UK to enable joined-up service delivery
+        paragraph: Analyse trends and patterns across whole journeys to help improve user experience
         steps:
         - heading: Recently shipped
           items:
-          - text: Create draft reports to see how services are being used, end-to-end
-          - text: Ongoing data analysis to help improve content and services related
-              to coronavirus and Brexit
-        - heading: Up next
+          - text: Expanded and automated our data reporting, including for coronavirus and Brexit content insights
+        - heading: Working on now
           items:
-          - text: Build data visualisation tools so it’s easier to analyse how anonymous
-              users interact with digital government services
-          - text: Automate the reports that show insights from coronavirus and Brexit
-              content
+          - text: Build tools to create anonymous data visualisations so it's easier to analyse how users interact with digital government services
+          - text: Make anonymous reports that show how users move across platforms and services, so we can improve their experience
+          - text: Get a better understanding of cookie opt-in rates, so we have a more accurate idea of actual visitor trends
         - heading: Exploring
           items:
-          - text: How to expand and automate data reports
-          - text: Further ways to automate analysis of user feedback so we can identify
-              trends more quickly
-      - heading: Everyone in the UK needs to understand how they are affected by coronavirus
-          and Brexit
-        paragraph: Create tools and guidance that help users find critical and up-to-date
-          information
+          - text: Find more ways to automate analysis so government can identify and respond to trends more quickly
+          - text: Experiment with audience analysis across sections of GOV.UK to understand page usage vs reading time to support product and content iteration
+          - text: Simplify how feedback is analysed and gathered, so we can better measure our impact and identify issues
+      - heading: Provide a personalised and proactive service
+        paragraph: Develop sitewide account features that offer users relevant information and more helpful notifications
```

Comment:
We spell `site-wide` without the hyphen in some places, and with the hyphen in other places, I would think it should be consistent throughout the page @keithiopia?

Reply:
I've gone with `site-wide`.

Reply:
Grammatically speaking, they both work. Good choice.